src/main/webapp/app/shared/user-settings/notification-settings/notification-settings.component.ts

```diff
@@ -21,6 +26,8 @@ export class NotificationSettingsComponent extends UserSettingsDirective impleme
     userSettings: UserSettingsStructure<NotificationSetting>;
     settings: Array<NotificationSetting>;
 
+    communicationChannel = NotificationSettingsCommunicationChannel;
```

Comment:
I'm unsure, is this line really needed here?

Reply:
I think this is to use the enum in the html template, e.g. when checking on equality.
I would make it a `readonly`, at least that is how I usually make enum types accessible in the html template.

Reply:
Yes exactly otherwise I can not access it in the HTML template. 
I added the readonly and a comment